src/Meshes/Meshes.jl

```diff
@@ -104,7 +104,12 @@ The element and face (`opelem`, `opface`) that oppose face `face` of element `el
 """
 function opposing_face end
 
+"""
+    Meshes.lengthscale(mesh::AbstractMesh)
 
+The approximate lengthscale of the elements of the mesh.
+"""
+function lengthscale end
```

Comment:
Is this the average element size? Can we call it something that recalls the element size then? Like `average_elem_size` or simply `elem_size`? I'm afraid lengthscale is not that commonly utilized and can be confused for a rescaling/nondimensionalization factor

Reply:
it is a rescaling/nondimensionalization factor: it's a length, not a size (e.g. the units are in metres).

Reply:
How about `elem_length` or `elem_lengthscale` then?

Reply:
I's prefer to spell it out: `element_length_scale` and `node_length_scale`?

Reply:
Thanks for clarifying. I indeed was confused by the meaning of it then. What is a `node_length_scale`? Nodes are points, i.e., 0-dimensional geometrical objects.

Reply:
it's the average distance between nodes, i.e. if there are `Nq` GLL nodes, then it is `element_length_scale / (Nq - 1)`

Reply:
Can we just call average node distance then? Sorry, I think it's important to make  these names self-explanatory

Reply:
In addition to this, I think it will also be important to add the min node distance to facilitate the CFL condition / Courant number calculation